cmd/vulcan-aws-trusted-advisor/main.go

```diff
@@ -227,11 +231,18 @@ func scanAccount(opt options, target string, logger *logrus.Entry, state state.S
 					logger.Debugf("resource with ResourceID: %s have been marked as excluded", *fr.ResourceId)
 					continue
 				}
-
-				header := []string{}
-				row := make(map[string]string)
+				// Get the alias of the account only if we did not get previously.
+				if alias == nil {
+					res, err := accountAlias(creds)
```

Comment:
If `accountAlias` depends only on `creds`, why not invoke this function right after getting the credentials, outside the main loop? For example, if you could initialise the alias on line 145, no? That way you don't need to care about wether you already have initialised the `alias` variable or not.

Also, another question, if for any reason we can not retrieve the account alias, should we fail the entire check execution?

Reply:
Good questions thanks for making them!!
- First question: I thought about it yes but finally, decided not do it because this way if an account does not have any issue we will save a request. But now that you mention it saving a request is not that much, so maybe, make more sense to declare where you said in order to make the code cleaner. What do you think?

- Second question:  To be honest I think yes it is better to make the  check fail. I mean, I understand that having the account alias can be hardly considered essential, but, I also think that the the call for getting alias should work almost always, so I preferred to make the check fail loudly in order to be aware that this call failed. In any case I agree that if we start seeing failures of the whole check for this reason we should change this.

Reply:
- Ahhhh, now I see your point! Indeed, it was smart to save a request. But also, it would be cleaner to initialise the alias outside the main loop. Now that you have clarified this to me, I would be ok with both implementations. Not strong preferences here.
- Understood and agree!

Thanks for the clarification. I will approve the PR as is, but feel free to modify it at will.